pkg/cli/policy/policy.go

```diff
@@ -21,7 +21,7 @@ func NewCmdPolicy(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobr
 	cmds.AddCommand(adminpolicy.NewCmdSccSubjectReview(f, streams))
 	cmds.AddCommand(adminpolicy.NewCmdSccReview(f, streams))
```

Comment:
It looks like we'll need to do the same thing for `oc adm policy scc-review` and `oc adm policy scc-subject-review`. They are still showing `oc policy` in their examples:

```
[ahoffer@ahoffer oc] (pr-973) $ ./oc adm policy scc-review --help
Check which service account can create a pod. The pod is inferred from the pod template spec in the provided resource.
If no service account is provided the one specified in podTemplateSpec.spec.serviceAccountName is used, unless it is
empty, in which case "default" is used. If service accounts are provided, the podTemplateSpec.spec.serviceAccountName is
ignored.

Usage:
  oc adm policy scc-review [flags]

Examples:
  # Check whether service accounts sa1 and sa2 can admit a pod with a template pod spec specified in my_resource.yaml
  # Service Account specified in myresource.yaml file is ignored
  oc policy scc-review -z sa1,sa2 -f my_resource.yaml
...
```
and 

```
[ahoffer@ahoffer oc] (pr-973) $ ./oc adm policy scc-subject-review --help
Check whether a user, service account or group can create a pod. It returns a list of security context constraints that
will admit the resource. If user is specified but not groups, it is interpreted as "what if the user is not a member of
any groups". If user and groups are empty, then the check is performed using the current user.

Usage:
  oc adm policy scc-subject-review [flags]

Examples:
  # Check whether user bob can create a pod specified in myresource.yaml
  oc policy scc-subject-review -u bob -f myresource.yaml
...
```

Reply:
Good catch - updated.